src/main/java/mate.academy/lib/Inject.java

```diff
@@ -1,5 +1,9 @@
 package mate.academy.lib;
 
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+@Retention(RetentionPolicy.RUNTIME)
```

Comment:
We can also mark @Target(ElementType=...) to determine kinds of elements an annotation interface can be applied to.